data/core/terrain.cfg

```diff
@@ -271,7 +271,7 @@
     symbol_image=flat/sandy-path
     id=road_desert
     name = _ "Road"
-    editor_name= _ "Rocky Sands"
+    editor_name= _ "Rocky Sand"
```

Comment:
Wondering if we should rename this completely. The stones aren't organised enough to be called "cobbled", but it's more like a dirt track that's been made more walkable by pushing stones in when it became muddy.

It doesn't look terrible next to desert, but it's notable that UtBS doesn't use this terrain. The only map in mainline that does is EI S04c (Mal Ravanals' Capital).

Reply:
Any issue with it being changed to Gravel now?